### DISCUSSION TITLE
Microsoft Translator

### DISCUSSION THREAD

**Discussion by qupaloraarsuk** [bitextor/bitextor#251] [General]
Does Bitextor accepts Microsoft Translator as a Machine Translation? We have a low-resource language (inuktitut) that is supported by MST, but not by other MT engines.

**Comment by ZJaume** [comment]
Translation is just a bash script that Bitextor calls expecting one output line per input line. So, if you are able to call Microsoft Translator or whatever translator, Bitextor will work with it.

**Comment by lpla** [comment]
We support any translation system that follows this I/O through `alignerCmd` setting in Bitextor:

`alignerCmd`: command to call the external MT script; the MT system must read documents (one sentence per line) from standard input, and write the translation, with the same number of lines, to standard output

(from https://github.com/bitextor/bitextor/blob/master/docs/CONFIG.md#using-external-mt)
